### PR TITLE
Add methods to write common type of messages as simple lines.

### PR DESCRIPTION
**Pull request by Aerendir**
| Q             | A
| ------------- | ---
| Branch?       | master 
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #21489 
| License       | MIT

Add method to write simple lines to the console.

- `info()` (fg: white; bg: blue): renders an info block
- `infoLine()` (fg: blue)
- `successLine()`
- `errorLine()`
- `warningLine()`
- `cautionLine()`

Refers to #21489.

### CODE REVIEW

File: src/Symfony/Component/Console/Style/StyleInterface.php [hunk 1]
```diff
@@ -47,27 +47,62 @@ public function listing(array $elements);
     public function text($message);
 
     /**
+     * Formats an info result bar.
+     *
+     * @param string|array $message
+     */
+    public function info($message);
```
Comment (stof) [inline src/Symfony/Component/Console/Style/StyleInterface.php]: Adding new methods in an interface is forbidden, as it breaks BC

File: src/Symfony/Component/Console/Style/SymfonyStyle.php [hunk 2]
```diff
@@ -151,6 +151,14 @@ public function success($message)
     /**
      * {@inheritdoc}
      */
+    public function successLine($message)
```
Comment (stof) [inline src/Symfony/Component/Console/Style/SymfonyStyle.php]: naming it ``*Line`` while it outputs a block looks weird to me

**Review by stof — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Console/Style/SymfonyStyle.php [hunk 3]
```diff
@@ -159,6 +167,14 @@ public function error($message)
     /**
      * {@inheritdoc}
      */
+    public function errorLine($message)
```
Comment (iltar) [inline src/Symfony/Component/Console/Style/SymfonyStyle.php]: What about calling them something like `Subtle`?
 - `errorSubtle`
 - `infoSubtle`
 - `warningSubtle`

File: src/Symfony/Component/Console/Style/StyleInterface.php (hunk 1 above)
Comment (Aerendir) [inline src/Symfony/Component/Console/Style/StyleInterface.php]: Yes, in fact I've seen that some other methods miss a correspondent mention in the interface... Ok, I'll go to remove them from the interface..

File: src/Symfony/Component/Console/Style/SymfonyStyle.php (hunk 2 above)
Comment (Aerendir) [inline src/Symfony/Component/Console/Style/SymfonyStyle.php]: I completely agree...

File: src/Symfony/Component/Console/Style/SymfonyStyle.php (hunk 3 above)
Comment (Aerendir) [inline src/Symfony/Component/Console/Style/SymfonyStyle.php]: I'll change their name...

Maybe to get what I'm trying to accomplish I should edit `OutputFormatter`...

**Pull request closed without merging**